unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/BuilderInWorldLoadingView.cs

```diff
@@ -1,35 +1,50 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
-using TMPro;
 using UnityEngine;
 using UnityEngine.UI;
 
 public interface IBuilderInWorldLoadingView
 {
-    event System.Action OnCancelLoading;
+    event Action OnCancelLoading;
 
-    void Show(bool showTips = true);
-    void Hide(bool forzeHidding = false);
+    bool isActive { get; }
+
+    void Show();
+    void Hide(bool forzeHidding = false, Action onHideAction = null);
     void StartTipsCarousel();
     void StopTipsCarousel();
-    void CancelLoading();
+    void CancelLoading(DCLAction_Trigger action);
+    void SetPercentage(float newValue);
 }
 
 public class BuilderInWorldLoadingView : MonoBehaviour, IBuilderInWorldLoadingView
 {
     private const string VIEW_PATH = "BuilderInWorldLoadingView";
 
-    [SerializeField] internal List<string> loadingTips;
+    [Header("Loading Tips Config")]
+    [SerializeField] internal BuilderInWorldLoadingTip loadingTipPrefab;
+    [SerializeField] internal RectTransform loadingTipsContainer;
+    [SerializeField] internal ScrollRect loadingTipsScroll;
+    [SerializeField] internal List<BuilderInWorldLoadingTipModel> loadingTips;
     [SerializeField] internal float timeBetweenTips = 3f;
-    [SerializeField] internal TMP_Text tipsText;
-    [SerializeField] internal Button cancelButton;
+    [SerializeField] internal AnimationCurve animationTipsCurve;
+    [SerializeField] internal float animationTipsTransitionTime = 1f;
+
+    [Header("Other Config")]
+    [SerializeField] internal InputAction_Trigger cancelLoadingInputAction;
     [SerializeField] internal float minVisibilityTime = 1.5f;
+    [SerializeField] internal LoadingBar loadingBar;
+
+    public event Action OnCancelLoading;
 
-    public event System.Action OnCancelLoading;
+    public bool isActive => gameObject.activeInHierarchy;
```

Comment:
since this mark that the isActive the view itself I prefer the `gameObject.activeSelf` since it describe more accurate if it is active even if the parent is not

Reply:
Done in https://github.com/decentraland/explorer/pull/2288/commits/0b42b25b4cd69cadb3a09181cd92e8a21fdb27c1. Thanks!